.travis.yml

```diff
@@ -1,40 +1,26 @@
-language: c
+language: generic
```

Comment:
leaving this would *probably* be fine, but, in general I usually set the language to generic since there is no official Fortran support on Travis. Setting a language can make more changes to your environment, some of which can be unexpected